nemo_skills/prompt/few_shot_examples/__init__.py

```diff
@@ -15,11 +15,13 @@
 from nemo_skills.prompt.few_shot_examples.math import examples_map as examples_math
 from nemo_skills.prompt.few_shot_examples.lean4 import examples_map as examples_lean4
 from nemo_skills.prompt.few_shot_examples.mmlu_pro import examples_map as examples_mmlu_pro
+from nemo_skills.prompt.few_shot_examples.mmlu import examples_map as examples_mmlu
 
 examples_map = examples_gsm8k.copy()
 examples_map.update(examples_math)
 examples_map.update(examples_lean4)
 examples_map.update(examples_mmlu_pro)
-assert len(examples_map) == len(examples_gsm8k) + len(examples_math) + len(
-    examples_lean4
-) + len(examples_mmlu_pro), "Duplicate keys in examples!"
+examples_map.update(examples_mmlu)
+assert len(examples_map) == len(examples_gsm8k) + len(examples_math) + len(examples_lean4) + len(
```

Comment:
can you please split this into multiple expressions by adding a variable for the length? Or some other way, the current assert is really hard to read with its weird three-line formatting

Reply:
Done